packages/govuk-frontend/src/govuk/components/character-count/character-count.mjs

```diff
@@ -62,6 +63,18 @@ export class CharacterCount extends GOVUKFrontendComponent {
   /** @private */
   maxLength = Infinity
 
+  /** @private */
+  configSchema = {
+    anyOf: [
```

Comment:
To honour "at least one, but not both" I think this should be `oneOf`?

```suggestion
    oneOf: [
```

Reply:
The way I read it, it IS currently valid to provide both - one will just be ignored.

Reply:
Yeah in the code it is, but did we mean that?? 😬

Reply:
Yours is definitely the sensible take, but I think to enforce it would be a breaking change, since services might have both properties right now, and we should obviously change the property descriptions.

Basically, do we need eyes from a non-dev? @romaricpascal, whaddya think?

Reply:
Looking at the documentation you're right, breaking change:
https://frontend.design-system.service.gov.uk/javascript-api-reference/#charactercount

### maxlength
> The maximum number of characters. If `maxwords` is provided, the `maxlength` option will be ignored.

### maxwords
> The maximum number of words. If `maxwords` is provided, the `maxlength` option will be ignored.

Reply:
Resolved in https://github.com/alphagov/govuk-frontend/pull/4183

Reply:
Yup, let's match the docs and keep `anyOf` as both could be set and we deal with it OK 😊 Nice little tidy up in #4183, makes both types and code simpler.